app/controllers/cases_controller.rb

```diff
@@ -799,22 +919,6 @@ def filter_params
     )
   end
 
-  def search_and_filter(full_list_of_cases = nil)
```

Comment:
This was the unused dormant method right?

Reply:
Yes this is the unused method that doesn't work